yt_dlp/extractor/floatplane.py

```diff
@@ -50,37 +50,30 @@ def _real_extract(self, url):
             media_id = media['id']
             media_typ = media.get('type') or 'video'
 
-            metadata = self._download_json(
-                f'{self._BASE_URL}/api/v3/content/{media_typ}', media_id, query={'id': media_id},
-                note=f'Downloading {media_typ} metadata', impersonate=self._IMPERSONATE_TARGET)
-
             stream = self._download_json(
-                f'{self._BASE_URL}/api/v2/cdn/delivery', media_id, query={
-                    'type': 'vod' if media_typ == 'video' else 'aod',
-                    'guid': metadata['guid'],
-                }, note=f'Downloading {media_typ} stream data',
+                f'{self._BASE_URL}/api/v3/delivery/info', media_id,
+                query={'scenario': 'onDemand', 'entityId': media_id},
+                note=f'Downloading {media_typ} stream data',
                 impersonate=self._IMPERSONATE_TARGET)
 
-            path_template = traverse_obj(stream, ('resource', 'uri', {str}))
+            metadata = self._download_json(
+                f'{self._BASE_URL}/api/v3/content/{media_typ}', media_id, query={'id': media_id},
+                note=f'Downloading {media_typ} metadata', fatal=False,
+                impersonate=self._IMPERSONATE_TARGET)
```

Comment:
annoying formatting comment
```suggestion
                f'{self._BASE_URL}/api/v3/content/{media_typ}', media_id,
                f'Downloading {media_typ} metadata', query={'id': media_id},
                fatal=False, impersonate=self._IMPERSONATE_TARGET)
```